ui/main.qml

```diff
@@ -81,6 +81,7 @@ ApplicationWindow {
         property real volume: 0.2
         property int notificationSetting: 0
         property bool notificationSoundsEnabled: true
+        property bool notifyNoneContactMessages: false
```

Comment:
This setting name is very confusing and that is part because there is a typo. It should be `notifyNonContactMessages`.
Maybe a name like `allowNotificationsFromNonContacts` would be more explicit?

Reply:
Absolutely.